packages/babel-plugin-transform-function-name/test/fixtures/function-name/unicode-encoding/options.json

```diff
@@ -0,0 +1,4 @@
+{
+    "plugins": ["transform-unicode-escapes"],
+    "presets": ["env"]
```

Comment:
```suggestion
    "plugins": ["transform-function-name", "transform-shorthand-properties"],
```

Reply:
Okay, I have changed the same in recent commit.